index.js

```diff
@@ -1,31 +1,25 @@
 const inquirer = require('inquirer')
 const fs = require('fs')
 const schema = require('./schema.json')
-const handleArray = require('./handleArray.js')
-const handleObject = require('./handleObject.js')
+const generateQuestions = require('./generateQuestions.js')
 
-// The program will loop through the imported schema object and call the appropriate function based on the type of each key
-for( const property in schema) {
-    if( typeof schema[property] === 'object') {
-        Array.isArray(schema[property]) ? handleArray(property, schema[property]) : handleObject(property, schema[property])
-    }
-}
+// 1. The first step is the generate our list of questions using the imported schema
+const questions = generateQuestions(schema)
 
-// inquirer
-//   .prompt([
-//     {
-//         name: 'test',
-//         message: 'This is a test, does it work?'
-//     }
-//   ])
-//   .then(answers => {
-//     if(answers.test === 'yes') console.log("It works!")
-//     else console.log("It still works")
-//   })
-//   .catch(error => {
-//     if(error.isTtyError) {
-//       // Prompt couldn't be rendered in the current environment
-//     } else {
-//       // Something else when wrong
-//     }
-//   });
+// 2. Next we use the inquirer package to prompt the user for the answers
```

Comment:
Great use of comments to list steps/explain your code for reviewers to follow along easily!

Reply:
Thank you!